modules/ROOT/pages/policy-mule4-jwt-validation.adoc

```diff
@@ -103,7 +109,7 @@ Ignore this field if you selected *none* as JWT Signing Method.
 
 | JWT Key Origin
 | Specifies where to obtain the key for the Signature validation. +
-You can provide the key in the policy selecting the *Text* option or obtain it from a JWKS. +
+You can provide the key in the policy selecting the *Text* option or obtain it from a *JWKS*. +
```

Comment:
```suggestion
You can provide the key in the policy by selecting the *Text* option or by obtaining it from a JWKS. +
```

Reply:
Question: if JWKS = ...Sets, plural, can we actually refer to "a" JWKS? Is it supposed to be Set, singular?

I see both when I search:

JSON Web Key Sets - Auth0https://auth0.com › Docs › Tokens › JSON Web Tokens
JSON Web Key Sets ... The JSON Web Key Set (JWKS) is a set of keys containing the public keys used to verify any JSON Web Token (JWT) issued by the ...
‎JSON Web Key Set Properties · ‎Locate JSON Web Key Sets · ‎Signing Keys

Reply:
Done! Thanks!